bin/reth/src/cli/ext.rs

```diff
@@ -46,6 +48,24 @@ pub trait RethNodeCommandConfig: fmt::Debug {
         Ok(())
     }
 
+    /// Event hook called once the rpc server has been started.
+    fn on_rpc_server_started<Conf, Reth>(
+        &mut self,
+        config: &Conf,
+        components: &Reth,
+        rpc_components: &RethRpcComponents<'_, Reth>,
+        handles: eyre::Result<(RpcServerHandle, AuthServerHandle)>,
```

Comment:
this should be a new wrapper type that has rpcserver handle and auth server handle as fields